site/src/examples/omezarr/omezarr-demo.tsx

```diff
@@ -250,17 +248,17 @@ export function OmezarrDemo() {
                             </div>
                         </div>
                     </div>
-                    <div style={{ display: 'flex', flexDirection: 'column', gap: '8px' }}>
-                        <label htmlFor="omezarr-json-view">Selected Image Metadata:</label>
-                        <textarea
-                            id="omezarr-json-view"
-                            readOnly
-                            cols={100}
-                            rows={36}
-                            style={{ resize: 'none' }}
-                            value={omezarrJson}
-                        />
-                    </div>
+                </div>
+                <div style={{ display: 'flex', flexDirection: 'column', gap: '8px' }}>
```

Comment:
Just moved the metadata display below the viewer because we're limited on horizontal space now that there's a sidebar